app.js

```diff
@@ -36,18 +49,22 @@ function checkFileType(file, cb){
   }
 }
 
-// Initialize app
-const app = express();
+
 
 // EJS
-app.set('view engine', 'ejs');
+// app.set('view engine', 'html');
 
 // Set 'public' folder as static
 app.use(express.static('./public'));
 
-app.get('/', (req, res) => res.render('index'));
+// app.get('/', (req, res) => res.render('index'));
+
+app.get("/", function(req, res) {
+  res.sendFile(__dirname + "/index.html");
+});
 
-app.post('/upload', (req, res) => {
+app.post('./public', (req, res) => {
+  res.sendFile(__dirname + "/upload.html");
```

Comment:
Why are you sending a template before uploading anything? 

- Create a new `GET` route to serve the template
- When the user clicks on submit, send a `POST` request to this route